etl/storage/storage.go

```diff
@@ -338,3 +352,41 @@ func (s *storage) GetJetDropByID(id models.JetDropID) (models.JetDrop, error) {
 	err := s.db.Model(&jetDrop).Where("pulse_number = ? AND jet_id = ?", id.PulseNumber, id.JetID).Find(&jetDrop).Error
 	return jetDrop, err
 }
+
+func (s *storage) GetJetDropsByJetID(jetID string, fromJetDropID *models.JetDropID, jetDropIDGte, jetDropIDLte *models.JetDropID, limit int, offset int, sortByPnAsc bool) ([]models.JetDrop, int, error) {
+	var jetDrops []models.JetDrop
+	var total int64
+
+	q := s.db.Model(&jetDrops).Where(&models.JetDrop{JetID: jetID})
+
+	if fromJetDropID != nil {
+		q = q.Where("jet_id like ?", fmt.Sprintf("%s%%", fromJetDropID.JetID))
```

Comment:
Will this work for searching parents of jetID? We should write tests to check finding parents and children by jetID, probably with hardcoded jetID values in test data.